pkg/chains/chains.go

```diff
@@ -179,6 +191,18 @@ var (
 		CctxGateway: CCTXGateway_observers,
 	}
 
+	// SolanaTestnet is Solana testnet
```

Comment:
Seems the `Devnet` is a commonly used name for Solana devs (user, app developer) and `Testnet` is for Solana core contributors stress test recent release features. See this: https://docs.solanalabs.com/clusters/available

Reply:
Good point, I will rename to avoid confusion

Reply:
a843cc4cc